### PR TITLE
Add notes on how to create a release

### PR DESCRIPTION
**Pull request by steakunderscore**
## Description
Notes taken from v4.1.0 release

## Motivation and Context
To make it easier for the next release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,4 +47,4 @@ If you would like to reach out to the maintainers, come talk to us in the `#oaut
 
 ## Contributing
 
-Please see our [Contributing](CONTRIBUTING.md) guidelines.
+Please see our [Contributing](CONTRIBUTING.md) guidelines. For releasing see our guide to [release creation guide](RELEASE.md).
```
Comment (JoelSpeed) [inline README.md]: Got the word guide twice in quick succession
```suggestion
Please see our [Contributing](CONTRIBUTING.md) guidelines. For releasing see our [release creation guide](RELEASE.md).
```

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  * Use format `v4.1.0` for both the tag and title
+2. Update [CHANGELOG.md](CHANGELOG.md)
+  * Write the release highlights
+  * Copy in headings ready for the next release
+3. Create release commit
+  ```
+  git checkout -b release-v4.1.0
+  ```
+4. Create pull request getting other maintainers to review
+5. Copy the release notes in to the draft Github release, adding a link to the [changelog](CHANGELOG.md)
```
Comment (JoelSpeed) [inline RELEASE.md]: The written `changelog` here is different to line 15, is that ok?

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+6. Update you local master branch
+  ```
+  git checkout master
+  git pull
+  ```
+7. Create & push the tag
+  ```
+  git tag v4.1.0
+  git push upstream --tags
```
Comment (JoelSpeed) [inline RELEASE.md]: `upstream` may not be accurate, eg if you had cloned the pusher fork as `origin` as I do, I'd be tempted to just make this `git push --tags`? 🤔

File: RELEASE.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  ```
+8. Make the release artefacts
+  ```
+  make release
+  ```
+9. Upload all the files (not the folders) from the `/release` folder to Github release as binary artefacts
```
Comment (JoelSpeed) [inline RELEASE.md]: Might be worth adding that these are tarballs and shasum text files?

File: RELEASE.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+10. Publish release in Github
+11. Make and push docker images to Quay
+  ```
+  make docker-all
+  make docker-push-all
+  ```
+  Note: Ensure the docker tags are tags aren't `-dirty`
```
Comment (JoelSpeed) [inline RELEASE.md]: Not 100% sure this makes sense?

File: RELEASE.md (hunk 2 above)
Comment (steakunderscore) [reply RELEASE.md]: It made sense at the time, but looking now I'll be consistent.

**Pull request merged**